tests/api/testdata/generators.py

```diff
@@ -87,6 +86,8 @@ def _fill_pytest_param(
     if endpoint.spec_link:
         marks.append(allure.link(url=endpoint.spec_link, name="Endpoint spec"))
     param_id = f"{endpoint.path.replace('/', '_')}_{method.name}_{positive_str}"
+    if sum(1 for e in Endpoints if e.path == endpoint.path) > 1:
+        param_id += f"_{endpoint.endpoint.get_data_class_pretty_name()}"
```

Comment:
probably you can move it to the endpoint, like something `endpoint.pretty_name()`
return path by default or path + dataclass if it's needed

Reply:
sure